modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -0,0 +1,151 @@
+= Choose an Iceberg Mode
+:description: Learn about supported Iceberg modes and how you can integrate schemas with Iceberg topics.
+:page-categories: Iceberg, Tiered Storage, Management, High Availability, Data Replication, Integration
+:schema-id-val-doc: manage:schema-reg/schema-id-validation.adoc
+// tag::single-source[]
+
+ifndef::env-cloud[]
+[NOTE]
+====
+include::shared:partial$enterprise-license.adoc[]
+====
+endif::[]
+
+In xref:manage:iceberg/about-iceberg-topics.adoc[Iceberg-enabled clusters], the `redpanda.iceberg.mode` topic property determines how Redpanda maps topic data to the Iceberg table structure. You can have the generated Iceberg table match the structure of an Avro or Protobuf schema in the Schema Registry, or you can use the `key_value` mode where Redpanda stores the record values as-is in the table. 
```

Comment:
Would it help readers more to link to https://deploy-preview-1068--redpanda-docs-preview.netlify.app/current/manage/iceberg/about-iceberg-topics/#enable-iceberg-integration instead here in this first line? Takes you to the context where you can view what, exactly, an iceberg-enabled config consists of, no?

Reply:
Also, can you make `redpanda.iceberg.mode` a link to the description of this property?

Reply:
Updated. Topic property page does not yet exist in Cloud docs